modules/release-notes/pages/relnotes.adoc

```diff
@@ -5,6 +5,254 @@ It also includes enhancements to the Eventing and Search services.
+|===
+
+==== Cross Datacenter Replication (XDCR)
+
+[#table_fixedissues_v602-xdcr,cols="25,66"]
+|===
+| Issue | Description
+
+| https://issues.couchbase.com/browse/MB-33970[MB-33970^]
+| *Summary*: By default, XDCR will prevent replications from being created on a source ephemeral bucket with NRU policy. This release of Couchbase Server introduces a special XDCR internal settings flag, `AllowSourceNRUCreation`, that allows replications to be created on a source ephemeral bucket with NRU policy. 
+
+----
+curl -X POST -u Administrator:password http://127.0.0.1:9998/xdcr/internalSettings -d AllowSourceNRUCreation="true"
+----
+
+WARNING: Changing the internal replication setting will cause all XDCR processes to restart. This causes temporary disruption as all existing replications will have to restart.
+
+| https://issues.couchbase.com/browse/MB-33671[MB-33671^]
+| *Summary*: Following an issue on a remote cluster, XDCR failed to recover and threw an exception.
+
+| https://issues.couchbase.com/browse/MB-33582[MB-33582^]
+| *Summary*: After upgrading to 6.0.x, it was observed that the compression type not set automatically. This has been fixed.
+
+| https://issues.couchbase.com/browse/MB-32814[MB-32814^]
+| *Summary*: Fixed REST endpoint error handling for settings errors and nil value types.
+|===
+
+==== Data Service
+
+[#table_fixedissues_v602-kv-data,cols="25,66"]
+|===
+| Issue | Description
+
+| https://issues.couchbase.com/browse/MB-34507[MB-34507^]
+| *Summary*: Fixed an issue observed when attempting to modify via subdoc by reseting the datatype to XATTRs if the pre-expiry callback returns a non-empty response.
+
+| https://issues.couchbase.com/browse/MB-34329[MB-34329^]
+| *Summary*: 
+
+| https://issues.couchbase.com/browse/MB-34173[MB-34173^]
+| *Summary*: Fixed an issue where recovery failed due to invalid snapshot start (or end) on disk. 
+
+| https://issues.couchbase.com/browse/MB-33919[MB-33919^]
+| *Summary*: The tombstone `delete-time` could be in the future or in the far past, causing the tombstone to be purged immediately or to remain on the disk for potentially long time. This has been fixed and the tombstone `delete-time` is now set to `ep_real_time` at the point of deletion.
+
+| https://issues.couchbase.com/browse/MB-33918[MB-33918^]
+| *Summary*: Fixed the cursor dropping trigger issue which caused Couchbase Server to get into a livelock state due to high checkpoint memory usage.
+
+| https://issues.couchbase.com/browse/MB-33854[MB-33854^]
+| *Summary*: Memcached returned ENGINE NOT_STORED to client after trying to perform arithmetic operations and caused a race condition.
+
+| https://issues.couchbase.com/browse/MB-33773[MB-33773^]
+| *Summary*: Fixed an issue where a chain of events, active compression of an extended attribute value followed by eviction and then expiry, triggered an exception.
+
+| https://issues.couchbase.com/browse/MB-33750[MB-33750^]
```

Comment:
This MB is related to the cluster manager (ns_server) - please move to the correct section.